spec/mail/message.mdown

```diff
@@ -258,15 +257,13 @@ If an id given cannot be found, the update or destroy MUST be rejected with a `n
 
 ### Saving a draft
 
-Creating messages via the *setMessages* method is only for creating draft messages and sending them. For delivering/importing a complete [@!RFC5322] message, use the `importMessages` method.
-
 The properties of the Message object submitted for creation MUST conform to the following conditions:
 
 - **id**: This property MUST NOT be included. It is set by the server upon creation.
 - **blobId**: This property MUST NOT be included. It is set by the server upon creation.
 - **threadId**: This property MUST NOT be included. It is set by the server upon creation.
-- **mailboxIds**: This property MUST be included. The value MUST include the id of either the mailbox with `role == "drafts"` (to save a draft) or the mailbox with `role == "outbox"` (to send the message). If this mailbox does not have `mustBeOnlyMailbox == true`, others may be included too.
-- **keywords**: This property MUST be included. It MUST include the `$Draft` keyword and SHOULD also include `$Seen`.
+- **mailboxIds**: This property MUST be included, and contain at least one mailbox.
+- **keywords**: This property MUST be included. If the message is a draft, it SHOULD include the `$Draft` and `$Seen` keywords.
```

Comment:
"If the message is a draft..." is a bit redundant given that the section name is "Saving a draft".

Reply:
Good catch. The title of the section should really be just Creating a message, but should reference saving a draft the primary example.